sensu/plugins/check-ceph-usage.rb

```diff
@@ -0,0 +1,150 @@
+#! /opt/sensu/embedded/bin/ruby
+#
+# check-ceph-usage
+#
+# DESCRIPTION:
+#   Raise alert if ceph cluster %RAW used exceed threshold
+#
+# OUTPUT:
+#   plain text, eixt code 0: OK, 1:Warning, 2:Critical, Others:Unknown
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   ceph client
+#
+# USAGE:
+#   #YELLOW
+#
+# NOTES:
+#   Runs 'ceph df' command(s) to report usage of ceph cluster. May 
+#   need read access to ceph keyring and/or root access for 
+#   authentication.
+#
+#   Using -z (--criticality) option to change criticality level. 
+#   if criticality is warning, raise warning alert; or raise critical
+#   alert.
+#
+#   Using -t (--threshold) option to determine alert level. 
+#   >= threshold, raise alert
+#
+# LICENSE:
+#   Copyright 2013 Brian Clark <brian.clark@cloudapt.com>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
+require 'sensu-plugin/check/cli'
+require 'timeout'
+require 'English'
+require 'json'
+
+class CheckCephClusterUsage < Sensu::Plugin::Check::CLI
+  option :keyring,
+         description: 'Path to cephx authentication keyring file',
+         short: '-k KEY',
+         long: '--keyring',
+         proc: proc { |k| " -k #{k}" }
```

Comment:
why the proc if the value does not need to transformed ?  I think this can be removed and the standard handling of the short version of the flag will suffice.